日 志 时 间:  2013-02-05 11:37:18,225 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:333)
日 志 内 容:  删除好友出错
 System.ServiceModel.FaultException: The server was unable to process the request due to an internal error.  For more information about the error, either turn on IncludeExceptionDetailInFaults (either from ServiceBehaviorAttribute or from the <serviceDebug> configuration behavior) on the server in order to send the exception information back to the client, or turn on tracing as per the Microsoft .NET Framework SDK documentation and inspect the server trace logs.

Server stack trace: 
   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannelProxy.InvokeEndService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   at Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(Object sender, DeleteFriendCompletedEventArgs e) in f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:line 279


日 志 时 间:  2013-02-05 11:37:24,480 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:333)
日 志 内 容:  删除好友出错
 System.ServiceModel.CommunicationObjectFaultedException: The communication object, System.ServiceModel.Channels.ServiceChannel, cannot be used for communication because it is in the Faulted state.

Server stack trace: 
   at System.Runtime.AsyncResult.End[TAsyncResult](IAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannel.SendAsyncResult.End(SendAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannelProxy.InvokeEndService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   at Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(Object sender, DeleteFriendCompletedEventArgs e) in f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:line 279


日 志 时 间:  2013-02-05 11:38:04,186 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:333)
日 志 内 容:  删除好友出错
 System.ServiceModel.FaultException: The server was unable to process the request due to an internal error.  For more information about the error, either turn on IncludeExceptionDetailInFaults (either from ServiceBehaviorAttribute or from the <serviceDebug> configuration behavior) on the server in order to send the exception information back to the client, or turn on tracing as per the Microsoft .NET Framework SDK documentation and inspect the server trace logs.

Server stack trace: 
   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannelProxy.InvokeEndService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   at Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(Object sender, DeleteFriendCompletedEventArgs e) in f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:line 279


日 志 时 间:  2013-02-05 11:43:19,976 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:333)
日 志 内 容:  删除好友出错
 System.ServiceModel.FaultException: The server was unable to process the request due to an internal error.  For more information about the error, either turn on IncludeExceptionDetailInFaults (either from ServiceBehaviorAttribute or from the <serviceDebug> configuration behavior) on the server in order to send the exception information back to the client, or turn on tracing as per the Microsoft .NET Framework SDK documentation and inspect the server trace logs.

Server stack trace: 
   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannelProxy.InvokeEndService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   at Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(Object sender, DeleteFriendCompletedEventArgs e) in f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:line 279


日 志 时 间:  2013-02-05 11:46:55,583 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:333)
日 志 内 容:  删除好友出错
 System.ServiceModel.FaultException: The server was unable to process the request due to an internal error.  For more information about the error, either turn on IncludeExceptionDetailInFaults (either from ServiceBehaviorAttribute or from the <serviceDebug> configuration behavior) on the server in order to send the exception information back to the client, or turn on tracing as per the Microsoft .NET Framework SDK documentation and inspect the server trace logs.

Server stack trace: 
   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannelProxy.InvokeEndService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   at Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(Object sender, DeleteFriendCompletedEventArgs e) in f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:line 279


日 志 时 间:  2013-02-05 11:48:30,812 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:333)
日 志 内 容:  删除好友出错
 System.ServiceModel.FaultException: The server was unable to process the request due to an internal error.  For more information about the error, either turn on IncludeExceptionDetailInFaults (either from ServiceBehaviorAttribute or from the <serviceDebug> configuration behavior) on the server in order to send the exception information back to the client, or turn on tracing as per the Microsoft .NET Framework SDK documentation and inspect the server trace logs.

Server stack trace: 
   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannelProxy.InvokeEndService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   at Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(Object sender, DeleteFriendCompletedEventArgs e) in f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:line 279


日 志 时 间:  2013-02-05 11:54:33,441 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:333)
日 志 内 容:  删除好友出错
 System.ServiceModel.FaultException: The server was unable to process the request due to an internal error.  For more information about the error, either turn on IncludeExceptionDetailInFaults (either from ServiceBehaviorAttribute or from the <serviceDebug> configuration behavior) on the server in order to send the exception information back to the client, or turn on tracing as per the Microsoft .NET Framework SDK documentation and inspect the server trace logs.

Server stack trace: 
   at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
   at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannelProxy.InvokeEndService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   at Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(Object sender, DeleteFriendCompletedEventArgs e) in f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:line 279


日 志 时 间:  2013-02-05 15:51:02,627 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 15:51:09,995 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 15:51:22,654 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 15:51:32,655 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 15:52:21,043 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 15:54:47,844 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 15:55:10,685 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 16:00:18,099 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 16:05:18,942 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 16:05:21,320 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 16:05:21,664 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 16:05:21,929 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 16:05:22,175 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 16:11:43,811 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:97)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 89


日 志 时 间:  2013-02-05 16:57:47,628 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:91)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 83


日 志 时 间:  2013-02-05 17:08:03,473 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:91)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 83


日 志 时 间:  2013-02-05 17:08:58,105 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:91)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 83


日 志 时 间:  2013-02-05 17:11:44,794 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:91)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 83


日 志 时 间:  2013-02-05 17:15:04,454 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:91)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 83


日 志 时 间:  2013-02-05 17:15:17,584 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:91)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 83


日 志 时 间:  2013-02-05 17:16:35,616 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:91)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 83


日 志 时 间:  2013-02-05 17:16:54,200 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:91)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 83


日 志 时 间:  2013-02-05 17:27:36,692 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:91)
日 志 内 容:  显示信息出错
 System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Chatter.MetroClient.DataUtil.SetCurrentMessageWindow(BaseRole baseRole) in f:\code\myRepo\Chatter\MetroClient\DataUtil.cs:line 141
   at Chatter.MetroClient.Callback.ChatterCallback.OnSendMessage(Message mesg) in f:\code\myRepo\Chatter\MetroClient\Callback\ChatterCallback.cs:line 83


日 志 时 间:  2013-02-05 17:32:16,891 
日 志 级 别:  ERROR
日 志 来 源:  Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:333)
日 志 内 容:  删除好友出错
 System.TimeoutException: This request operation sent to net.tcp://10.2.5.57:9877/ChatterService did not receive a reply within the configured timeout (00:01:10).  The time allotted to this operation may have been a portion of a longer timeout.  This may be because the service is still processing the operation or because the service was unable to send a reply message.  Please consider increasing the operation timeout (by casting the channel/proxy to IContextChannel and setting the OperationTimeout property) and ensure that the service is able to connect to the client.

Server stack trace: 
   at System.Runtime.AsyncResult.End[TAsyncResult](IAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannel.SendAsyncResult.End(SendAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)
   at System.ServiceModel.Channels.ServiceChannelProxy.InvokeEndService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   at Chatter.MetroClient.UI.MyButton.Client_DeleteFriendCompleted(Object sender, DeleteFriendCompletedEventArgs e) in f:\code\myRepo\Chatter\MetroClient\UI\MyButton.xaml.cs:line 279


